src/gps/gpsd_test.cpp

```diff
@@ -0,0 +1,136 @@
+/* Note from Nathan Hatch (Husky Robotics):
+ *
+ * This code is copied from https://github.com/jcable/gpsd/blob/master/test_gpsmm.cpp
+ * and modified slightly. (There was a change in how the gps_data_t struct stores
+ * the information about the satellites.)
+ */
+
+/*
+ * Copyright (C) 2010 Eric S. Raymond.
+ *
+ * This software is distributed under a BSD-style license. See the
+ * file "COPYING" in the top-level directory of the distribution for details.
+ *
+ */
+
+/* This simple program shows the basic functionality of the C++ wrapper class */
+#include <iostream>
+
+#include "libgpsmm.h"
```

Comment:
CMake seems to handle it fine either way but just for the sake of conventions I think this should use angle brackets to make it clear this is a system header and not a local one.